package.json

```diff
@@ -1,10 +1,12 @@
 {
   "name": "koa-webpack",
-  "version": "0.1.7",
+  "version": "0.1.8",
   "description": "Development and Hot Reload Middleware for Koa2",
   "main": "dist/index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "gulp build",
+    "lint": "gulp lint"
```

Comment:
I'm not sure how to run gulp scripts outside of having to install it as a global module. I don't use gulp so I don't have it installed globally, I thought this was the standard way of accessing it in that case. I'll remove it though.